libraries/cyclestreets-fragments/src/main/res/values-pt/dimens.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <dimen name="app_defaultsize_w">632.0dip</dimen>
+  <dimen name="app_defaultsize_h">598.0dip</dimen>
```

Comment:
This file, as well as `colors.xml`, `styles.xml`, `dimens.xml` doesn't contain text that can be localised, so there's no need to duplicate them in the `values-pt` directory.  (If the `-pt` directory doesn't contain a resource, the default is used.)

Please therefore remove any untranslated files from your PR.